src/main/kotlin/net/ccbluex/liquidbounce/render/RenderShortcuts.kt

```diff
@@ -113,6 +114,18 @@ fun RenderEnvironment.withColor(color4b: Color4b, draw: RenderEnvironment.() ->
     RenderSystem.setShaderColor(1f, 1f, 1f, 1f)
 }
 
+/**
+ * Extension function to disable cull
+ * Good for rendering faces that should be visible from both sides
+ *
+ * @param draw The block of code to be executed with cull disabled.
+ */
+fun RenderEnvironment.withDisabledCull(draw: RenderEnvironment.() -> Unit) {
+    RenderSystem.disableCull()
+    draw()
+    RenderSystem.enableCull()
+}
```

Comment:
An Exception in this area would leave cull disabled. It would be best if it was something like `try { draw() } finally { enableCull() }` so the cull will be disabled regardless if an exception occured or not

Reply:
I have also added it to the withPosition function and withColor function, because the same problem seemed to apply to them.